extension.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "SemanticMediaWiki",
+	"version": "3.0.0-alpha",
+	"author": [
+        "[https://www.mediawiki.org/wiki/User:Jeroen_De_Dauw Jeroen De Dauw]",
+        "James Hong Kong",
+        "[http://korrekt.org Markus Krötzsch]",
```

Comment:
I am more happy with the old ordering. Markus definitely did more than me so I prefer to be listed after him.